website/static/js/forgotPassword.js

```diff
@@ -7,37 +7,61 @@ var ko = require('knockout');
 require('knockout-validation');
 
 var $osf = require('js/osfHelpers');
+var oop = require('js/oop');
+var formViewModel = require('js/formViewModel');
 
-var ViewModel = function() {
 
-    var self = this;
+var ForgotPasswordViewModel = oop.extend(formViewModel.FormViewModel, {
+    constructor: function() {
+        var self = this;
+        self.super.constructor();
+        self.username = ko.observable('').extend({
+            required: true,
+            email: true
+        });
+    },
+    isValid: function() {
+        var ValidationError = new formViewModel.ValidationError();
+        if (!this.username.isValid()) {
+            ValidationError.messages.push('Please enter a valid email address.');
+            throw ValidationError;
+        } else {
+            return true;
+        }
+    }
+});
 
+var ForgotPasswordViewModel = function() {
```

Comment:
This is a redefinition of ForgotPasswordViewModel--was this meant to be removed?